Source/FakeItEasy/Creation/IFakeOptionsBuilder.cs

```diff
@@ -66,6 +66,16 @@ public interface IFakeOptionsBuilder<T>
         IFakeOptionsBuilder<T> Implements(Type interfaceType);
 
         /// <summary>
+        /// Sets up the fake to implement the specified interface in addition to the
+        /// originally faked class.
+        /// </summary>
+        /// <typeparam name="TInterface">The type of interface to implement.</typeparam>
+        /// <returns>Options object.</returns>
+        /// <exception cref="ArgumentException">The specified type is not an interface.</exception>
+        [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "Implements", Justification = "Would be a breaking change, might be changed in a later major version.")]
+        IFakeOptionsBuilder<T> Implements<TInterface>();

```

Comment:
Another optional warning, if you're inclined to fix these kinds of things. The Code Analysis complains about the unused `TInterface`. A suppression like the one on `IAnyCallConfigurationWithNoReturnTypeSpecified.IAnyCallConfigurationWithReturnTypeSpecified` would clean this up:

`[SuppressMessage("Microsoft.Design", "CA1004:GenericMethodsShouldProvideTypeParameter", Justification = "Used to provide a strongly typed fluent API.")]`

or similar.